src/components/dialogs/upload-case.js

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import { useDispatch, useSelector } from 'react-redux';
+import { selectFile } from '../../redux/actions';
+import Button from '@material-ui/core/Button';
+import { FormattedMessage } from 'react-intl';
+import React from 'react';
+
+export const UploadCase = () => {
+    const dispatch = useDispatch();
+    const selectedFile = useSelector((state) => state.selectedFile);
+
+    const handleFileUpload = (e) => {
+        e.preventDefault();
+        let files = e.target.files;
+        dispatch(selectFile(files[0]));
```

Comment:
We don't manage multiple imports ? Maybe it's not that hard... take a look...

Reply:
how do we manage name ?